docs/book/introduction/quickstart-guide.md

```diff
@@ -74,33 +79,32 @@ def trainer(
         metrics=["accuracy"],
     )
 
-    model.fit(X_train, y_train)
+    model.fit(x_train, y_train)
 
-    # write model
     return model
 
 
 @step
 def evaluator(
-    X_test: np.ndarray,
```

Comment:
I actually really liked the capitalized `X_test` for X matrices. I think its a convention (or atleast that how I learned it)

Reply:
Alright, I can return it to capitalized